src/display.rs

```diff
@@ -150,6 +150,13 @@ impl Display {
         // Create renderer
         let mut renderer = QuadRenderer::new(viewport_size)?;
 
+        // TODO: replace `set_position` with `with_position` once available
+        // Set window position early so it doesn't "teleport"
+        let position = options.position().or_else(|| config.position());
+        if let Some(position) = position {
+            window.set_position(position.x, position.y);
```

Comment:
Is there any reason why the position isn't set right after window initialization?

Reply:
Moved right under `let mut window`. Hopefully that's what you meant.